.github/workflows/lint.yml

```diff
@@ -342,7 +342,7 @@ jobs:
       - name: SQLFluff
```

Comment:
Can we be more descriptive about what this step is, similar to the ones around it?